src/m_misc.c

```diff
@@ -2273,31 +2276,40 @@ default_t defaults[] = {
   {
     "midi_player",
     (config_t *) &midi_player, NULL,
-    {0}, {0, MAX_MIDI_PLAYERS},
+    {0}, {0, 2},
     number, ss_gen, wad_no,
 #if defined(_WIN32)
-    "MIDI player"
+    "0 for Native (default), "
 #else
     "0 for SDL2 (default), "
-  #if defined(HAVE_FLUIDSYNTH)
+#endif
+#if defined(HAVE_FLUIDSYNTH)
     "1 for FluidSynth, 2 for OPL Emulation"
-  #else
+#else
     "1 for OPL Emulation"
-  #endif
 #endif
   },
 
+  {
```

Comment:
Do we need to expose this to the config system if we already have `midi_player` and `soundfont_path`/`winmm_device`? Could we reconstruct `midi_player_menu` from these three?

Reply:
But can we add a menu item without exposing it in the config?

Reply:
Oops, good question!